File: June2/Seq.java
public class Seq
{
    static int isSeq(int arr[])
    {
        int n=arr.length;
        int sum=n*(n+1)/2;
        for(int i=0;i<n;i++)
        {
            if(arr[i]>0&&arr[i]<n)
                sum-=arr[i];
        }
        return sum;
    }

    public static void main(String args[])
    {

        int a[]={2,3,-1,1};
        System.out.println(isSeq(a));
    }

}

File: June2/seq.cpp
#include<iostream>
#include<vector>
using namespace std;
int missingSeq(vector <int> &arr)
{
    int n=arr.size();
    int sum=n*(n-1)/2;
    for(int i=0;i<n;i++)
    {
        if(arr[i]>0&&arr[i]<n)
            sum-=arr[i];
    }

    return sum;
    

}

int main()
{
vector<int> a={3,4,-1,1};
int res=missingSeq(a);
cout<<res;

}